config/services.yaml

```diff
@@ -31,6 +33,8 @@ services:
             $tablePrefix: '%bolt.table_prefix%'
             $backendUrl: '%bolt.backend_url%'
             $rememberLifetime: '%bolt.remember_lifetime%'
+            $translatableFetchMode: '%bolt.doctrine_behaviors_translatable_fetch_mode%'
+            $translationFetchMode: '%bolt.doctrine_behaviors_translation_fetch_mode%'
```

Comment:
```suggestion
```

I do not think these two need a default bind. They are only used in the listeners, and should be explicitly bound to that service.